_scripts/deploy.sh

```diff
@@ -6,7 +6,7 @@
 cd "$(dirname "$0")" || exit 1
 
 export IMAGE_PREFIX=deisci BUILD_TAG=v2-beta
-docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-DEIS_REGISTRY='' make -C .. build push
+#docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
@mboersma Gotcha. I made https://github.com/deis/monitor/issues/40 for that purpose - I just need help getting the creds for dockerhub and ill update this.